src/tests/system/tests/test_tools.py

```diff
@@ -0,0 +1,61 @@
+"""
+Tools Tests.
+
+Tests pertaining to command line tools, some tools will have their own file.
+
+* sssctl: test_sssctl.py
+* sss_cache
+* sss_obfuscate
+* sss_seed
+* sss_debuglevel
+* sss_override: sss_override.py
+* sss_ssh_authorizedkeys
+* sss_ssh_knownhostsproxy
+
+:requirement: Tools
+"""
+
+from __future__ import annotations
+
+import pytest
+from pytest_mh.ssh import SSHProcessError
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.importance("medium")
+@pytest.mark.ticket(bz=1661182)
+@pytest.mark.topology(KnownTopology.Client)
+def test_tools__sss_cache_expired_does_not_print_unrelated_message(client: Client):
```

Comment:
I wonder if this test is really needed, but it's your call.

Reply:
There is a bz associated with it. Personally, I don't find log assertion tests important. For right now, since there is a bz, let's just hang on to it.